apps/dashboard/app/models/project.rb

```diff
@@ -90,13 +90,25 @@ def initialize(attributes = {})
 
   def to_h
     {
-      :id => id,
-      :name => name,
+      :id          => id,
+      :name        => name,
       :description => description,
-      :icon => icon,
+      :icon        => icon,
     }
   end
 
+  def to_json(options = {})
```

Comment:
OK - I guess I have something more meaty than a nitpick - We should probably be using the `jbuilder` utilities like we use elsewhere. 

Here you can see the view the files controller/models use when responding to json. 

https://github.com/OSC/ondemand/blob/master/apps/dashboard/app/views/files/index.json.jbuilder

Reply:
Made some changes based on the feedback and fixed the failing tests.